### ISSUE TITLE
Validation below training curve in Learning Curve

### ISSUE THREAD

**Issue by do2do** [do2do/BankCrisis#2]
### Likely due to difference in positive rate between training and validation sample.
Temporary fix put by using a different random seed during shuffle
Could use TF to acheive stratified shuffling
Could also duplicate crisis samples on the training set

**Comment by do2do** [comment]
### A useful resource found by @johnsonice 👍 
[https://machinelearningmastery.com/tactics-to-combat-imbalanced-classes-in-your-machine-learning-dataset/](url)

**Comment by do2do** [comment]
#### ***Note: Training set below validation set in learning curve with Feature Engineering***

## Training Set (minuscule improvement, unstable)
### Before: 
{'accuracy': 0.9561667,
 'accuracy_baseline': 0.9575,
 'auc': 0.6920112,
 'auc_precision_recall': 0.08453596,
 'average_loss': 0.1705845,
 'global_step': 500,
 'label/mean': 0.0425,
 'loss': 0.1705845,
 'precision': 0.16666667,
 'prediction/mean': 0.04297705,
 'recall': 0.007843138}
### After:

{'accuracy': 0.9571667,
 'accuracy_baseline': 0.9575,
 'auc': 0.699584,
 'auc_precision_recall': 0.10159188,
 'average_loss': 0.16622697,
 'global_step': 500,
 'label/mean': 0.0425,
 'loss': 0.16622697,
 'precision': 0.0,
 'prediction/mean': 0.036719635,
 'recall': 0.0}

## Validation Set (large deterioration):
### Before:
{'accuracy': 0.9585742,
 'accuracy_baseline': 0.9585742,
 'auc': 0.6824588,
 'auc_precision_recall': 0.074727386,
 'average_loss': 0.16871664,
 'global_step': 500,
 'label/mean': 0.04142582,
 'loss': 0.16871664,
 'precision': 0.0,
 'prediction/mean': 0.043120008,
 'recall': 0.0}

### After:
{'accuracy': 0.9585742,
 'accuracy_baseline': 0.9585742,
 'auc': 0.61071634,
 'auc_precision_recall': 0.09101391,
 'average_loss': 0.17415352,
 'global_step': 500,
 'label/mean': 0.04142582,
 'loss': 0.17415352,
 'precision': 0.0,
 'prediction/mean': 0.03298234,
 'recall': 0.0}
